quickdraw/compilation_equivalence.test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Dir["./compilation_equivalence_cases/*.rb", base: File.dirname(__FILE__)].each do |file|
+	test File.basename(file) do
+		load File.expand_path(file, File.dirname(__FILE__))
+
+		class_name = File.basename(file, ".rb").split("_").map(&:capitalize).join
+		component = Object.const_get(class_name)
+
+		before = component.new.call
+		Phlex::Compiler.compile(component)
+		after = component.new.call
+
+		assert_equal after, before
```

Comment:
Should we use `assert_equal_html` here?

Reply:
Yes we should!

Reply:
![CleanShot 2025-07-08 at 13 20 24@2x](https://github.com/user-attachments/assets/e192ebe0-a93c-452f-8055-d4073ffe1c11)
That's better!

Reply:
The reason this isn’t working is we haven’t got support for void elements yet.